Calico/Vpp docs cleanup (#272)
tigera/docs: calico/reference/vpp/uplink-configuration.mdx

## General mechanics

> The `calico-vpp-config` ConfigMap section of the manifest yaml contains the key `CALICOVPP_INTERFACES` which is a dictionary with parameters

Reviewer: ```suggestion
The `calico-vpp-config` ConfigMap section of the manifest file contains the key `CALICOVPP_INTERFACES`, which is a dictionary with parameters
```

Author: Done. Thanks for catching it.

:::

> - For the primary interface, ie, `uplinkInterfaces[0]`, set `vppDriver` as "af_xdp"

Reviewer: ```suggestion
- For the primary interface, `uplinkInterfaces[0]`, set `vppDriver` to "af_xdp"
```

Author: Done for all the occurrences of similar lines for the other drivers too. Thanks!

- Also ensure that your vpp config has no `dpdk` stanza and its plugin disabled
- Finally `FELIX_XDPENABLED` should be set to `false` on the `calico-node` container otherwise felix will periodically cleanup the VPP configuration

```yaml

> ---

Reviewer: Should this line be removed too?

Author: This already removed I think.
